app/src/pages/trace/TracePage.tsx

```diff
@@ -572,6 +579,98 @@ function RetrieverSpanInfo(props: {
   );
 }
 
+function RerankingSpanInfo(props: {
+  span: Span;
+  spanAttributes: AttributeObject;
+}) {
+  const { spanAttributes } = props;
+  const rerankingAttributes = useMemo<AttributeObject | null>(() => {
+    const rerankingAttrs = spanAttributes[SemanticAttributePrefixes.reranking];
+    if (typeof rerankingAttrs === "object") {
+      return rerankingAttrs as AttributeObject;
+    }
+    return null;
+  }, [spanAttributes]);
+  const query = useMemo<string>(() => {
+    if (rerankingAttributes == null) {
+      return "";
+    }
+    return (rerankingAttributes[RerankingAttributePostfixes.query] ||
+      "") as string;
+  }, [rerankingAttributes]);
+  const input_documents = useMemo<AttributeDocument[]>(() => {
+    if (rerankingAttributes == null) {
+      return [];
+    }
+    return (rerankingAttributes[RerankingAttributePostfixes.input_documents] ||
+      []) as AttributeDocument[];
+  }, [rerankingAttributes]);
+  const output_documents = useMemo<AttributeDocument[]>(() => {
+    if (rerankingAttributes == null) {
+      return [];
+    }
+    return (rerankingAttributes[RerankingAttributePostfixes.output_documents] ||
+      []) as AttributeDocument[];
+  }, [rerankingAttributes]);
+
+  const numInputDocuments = input_documents.length;
+  const numOutputDocuments = output_documents.length;
+  return (
+    <Flex direction="column" gap="size-200">
+      <Card title="Query" {...defaultCardProps}>
+        <CodeBlock value={query} mimeType="text" />
+      </Card>
+      <Card
+        title={`Input Documents (${numInputDocuments})`}
+        {...defaultCardProps}
+        defaultOpen={false}
+      >
+        {
+          <ul
+            css={css`
+              padding: var(--ac-global-dimension-static-size-200);
+              display: flex;
+              flex-direction: column;
+              gap: var(--ac-global-dimension-static-size-200);
+            `}
+          >
+            {input_documents.map((document, idx) => {
+              return (
+                <li key={idx}>
+                  <DocumentItem document={document} />
```

Comment:
I might re-color these just so that there's a visual hierarchy of color (e.g. that re-ranked documents take on a different tint) - this way as you are clicking around you can clearly see the difference.

Reply:
Re-using the `DocumentItem` component is good but I think just showing the new `score` label might be a tad confusing? Or just using `score` as an abstract is intended here.

Reply:
In one case it's often spacial distance where as when running through a reranker it is a relevance rank. Just thinking that from a user's perspective displaying `score: XX` alongside both we lose a bit of an opportunity to explain the `score` in this context a bit better - `score` being pretty generic.

Reply:
I think even though `score` is generic, it is still accurate. On the input side of the reranker, `score` may or may not exist, and even if it does exist, it's not considered by the reranker. But if the "input" `score` does exist it was generated by a preprocessor for a separate purpose. The general mental picture here is that there could be millions of documents in a corpus, and only a relatively small set are chosen to be reranked, and that selection process can have a `score` of its own based on the query in question. Even though that `score` is not meaningful to the reranker, it is still an informative attribute of the input document, because it relays the reason for how the document became a candidate in the first place (especially when the preprocessor is missing in the trace). On the other hand, we can't really get more specific that the `score` verbiage because we don't have more information. On balance, although it may seem confusing at first, a user should have enough context to reason their way through it.

Reply:
I think I wasn't disputing the way we capture the score - was just thinking of ways to avoid the mental "eason their way through it." a bit. But I don't have an immediate good prefix for the reranker score so let's keep it for now.